detection-rules/vip_impersonation_attack_surface_reduction.yml

```diff
@@ -0,0 +1,61 @@
+name: "VIP / Executive impersonation (first-time sender, unsolicited)"
+description: |
+    Sender display name matches the display name of a user in the $org_vips list,
+    and the sender has never been seen before.
+
+    The $org_vips list must be manually connected to a VIP group of your upstream
+    provider (Google Workspace and Microsoft 365 only) in order for this rule to work.
```

Comment:
worth noting the list will automatically be kept up-to-date after manually connecting